src/tracker.js

```diff
@@ -12,26 +13,65 @@ const defaultPost = (url, body) => fetch(url, {
   body: JSON.stringify(body)
 });
 
+export const EventType = {
+  PAGE_VISITED: 'PAGE_VISITED',
+  LINK_CLICKED: 'LINK_CLICKED',
+  FORM_SUBMITTED: 'FORM_SUBMITTED'
+};
+
+const URL = t.maybe(t.String);
+const NotEmptyString = t.refinement(t.String, s => s.length > 0);
+
+const postEvent =
+  ({url, application, platform, browser}, post, log) =>
```

Comment:
what do you think about defining ourselves the browser value, with a default value, and use it as the default parameter for the browser property ?